tests/TestHarness/launch_transaction_generators.py

```diff
@@ -11,6 +11,8 @@
 sys.path.append(harnessPath)
 
 from .testUtils import Utils
```

Comment:
Note calling launch_transaction at any level results in the following for me. Ubuntu 22, Python 3.10.6.
Works if I switch from `.testUtils` to `testUtils`, but I'm not sure that's correct?
```
root@e3e58d88e458:/mnt/leap/Build-22-gcc/tests/TestHarness# ./launch_transaction_generators.py -?
Traceback (most recent call last):
  File "/mnt/leap/Build-22-gcc/tests/TestHarness/./launch_transaction_generators.py", line 13, in <module>
    from .testUtils import Utils
ImportError: attempted relative import with no known parent package
```

Reply:
Since `launch_transaction_generators.py` is incorporated in the `TestHarness` module, the off-the-shelf way to run the script is: `python3 -m TestHarness.launch_transaction_generators`.  Running it as `./launch_transaction_generators.py` is officially unsupported.  There are some additional hacky ways to get it to work, but the mentioned methodology is preferred.  I have cleaned up the executable bits on the script to try to make that more apparent.